tests/automated_install.sh

```diff
@@ -20,7 +20,7 @@ if [[ "$(pwd)" != "$(git rev-parse --show-toplevel)" ]]; then
 fi
 
 export BACH_ENVIRONMENT='Test-Laptop'
-export BACH_CLUSTER_PREFIX='foo'
+export BACH_CLUSTER_PREFIX={BACH_CLUSTER_PREFIX:-'foo'}
```

Comment:
No need for foo here. That was an oversight in my commit. No cluster prefix (`''`) is what we used to do effectively. Ideally this should come from the environment anyhow as we need it in our Chef code and this must match the environment.